doc/plutus/troubleshooting.rst

```diff
@@ -64,28 +10,6 @@ To reduce code size, on-chain errors only output codes. Here's what they mean:
   This list can be generated with:
   grep -rEoh "\btrace\w*\s+\"[^\"]{1,5}\"\s+(--.*|\{-\".*\"-\})" *
 
-- Ledger errors
```

Comment:
This seems the wrong way around. The Ledger and state machine errors should stay. The prelude errors should go.

Reply:
OK, I've made that change. Should the Currency Errors also stay?

Reply:
Yes.